gui/gui.go

```diff
@@ -47,6 +53,7 @@ type GUI struct {
 	cookieStore *sessions.CookieStore
 	router      *mux.Router
 	server      *http.Server
+	certMgr     *autocert.Manager
```

Comment:
There isnt any need for this to be added to the GUI struct. It can be declared and remain local inside the func `Run()`

Reply:
Noted, updating.